pubspec.yaml

```diff
@@ -13,35 +13,35 @@ dependencies:
   flutter_localizations:
     sdk: flutter
 
-  app_review: 1.1.2
-  cupertino_icons: 1.0.0
+  app_review: ^2.1.1+1
+  cupertino_icons: ^1.0.2
   diff_match_patch: 0.3.0
-  flare_flutter: 2.0.6
-  flutter_compass: 0.4.3
-  flutter_map: 0.10.1+1
-  flutter_svg: 0.19.0
-  geolocator: ^6.1.4+1
-  google_api_availability: 2.0.4
-  http: 0.12.2
-  intl: 0.16.1
-  intl_translation: 0.17.10+1
-  mockito: 4.1.2
-  package_info: 0.4.3
-  path_provider: 1.6.18
-  permission_handler: 4.4.0+hotfix.4
-  rxdart: 0.24.1
-  share: 0.6.5+2
-  shared_preferences: 0.5.12
-  sprintf: 5.0.0
-  synchronized: 2.2.0+2
-  url_launcher: 5.7.2
+  flare_flutter: ^3.0.0
+  flutter_compass: ^0.6.0
+  flutter_map: ^0.12.0
+  flutter_svg: ^0.21.0+1
+  geolocator: ^7.0.1
+  google_api_availability: ^3.0.1
+  http: 0.13.1
+  intl: 0.17.0
+  mockito: 4.1.4
+  package_info: 2.0.0
+  path_provider: ^2.0.1
+  permission_handler: ^6.1.1
+  rxdart: ^0.26.0
+  share: ^2.0.1
+  shared_preferences: ^2.0.5
+  sprintf: ^6.0.0
+  synchronized: ^3.0.0
+  url_launcher: 6.0.3
   uuid: 2.2.2
 
 dev_dependencies:
   flutter_driver:
     sdk: flutter
   flutter_test:
     sdk: flutter
+  network_image_mock: 1.1.0
```

Comment:
This new dependency comes from https://github.com/stelynx/network_image_mock. It allows us to test the package. The benefit is already ready for Flutter Null Safety.